app/submitter/submitter.py

```diff
@@ -7,16 +8,33 @@
 logger = get_logger()
 
 
-class LogSubmitter():  # pylint: disable=no-self-use
+class LogSubmitter:
+    @staticmethod
+    def send_message(message, **kwargs):
+        logger.info('sending message')
+        logger.info('message payload', message=message, case_id=kwargs['case_id'], tx_id=kwargs['tx_id'])
+
+        return True
+
 
-    def send_message(self, message, queue, tx_id):  # pylint: disable=unused-argument
+class GCSSubmitter:
+
+    def __init__(self, project_id, bucket_name):
+        client = storage.Client(project_id)
+        self.bucket = client.get_bucket(bucket_name)
+
+    def send_message(self, message, **kwargs):
         logger.info('sending message')
-        logger.info('message payload', message=message, queue=queue)
+
+        blob = self.bucket.blob(kwargs['case_id'])
```

Comment:
I think we should consider putting some metadata on the object so we can identify what it is without having to decrypt

Reply:
Anything specific you had in mind?

Reply:
tx_id, which questionnaire maybe?

Reply:
Addressed, wasn't sure about key names. For now the format is:
```
{
  "metadata": {
    "tx_id": "744c0afc-6153-4047-b51b-28926990feca"
  },
  "submission": "encrypted_message"
}
```